docs/data/pages.ts

```diff
@@ -17,9 +17,10 @@ const pages: MuiPage[] = [
       {
         pathname: '/toolpad/examples',
         children: [
-          { pathname: '/toolpad/examples/npm-stats', title: 'npm stats' },
+          { pathname: '/toolpad/examples/npm-stats', title: 'NPM stats' },
```

Comment:
I suspect it's 

```suggestion
          { pathname: '/toolpad/examples/npm-stats', title: 'npm stats' },
```

per https://www.npmjs.com/